### ISSUE TITLE
Theme issues for all components in vanilla `create-react-app` project

### ISSUE THREAD

**Issue by tgroshon** [instructure/instructure-ui#15]
Very cool project with _great_ UI components!

The process to get started with _any components_ from this library is very difficult however because of the apparent requirement of every component to setup the entire `ui-themeable` build pipeline.

Is the *theming* process _meant_ to be a hard requirement of all these components?  If so, what's the *Quick Start* to get something simple working with a simple build like `create-react-app`?

Here's an example experiment using `Heading` from `@instructure/ui-core`:

```javascript
import React, { Component } from 'react';
import logo from './logo.svg';
import './App.css';

// Added @instructure-ui component import
import Heading from '@instructure/ui-core/lib/components/Heading';

class App extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <img src={logo} className="App-logo" alt="logo" />
          <h1 className="App-title">Welcome to React</h1>
        </header>
        <p className="App-intro">
          To get started, edit <code>src/App.js</code> and save to reload.
        </p>

        <Heading>Hello World!</Heading>
      </div>
    );
  }
}
export default App;
```

The result is a nasty theme generator stack trace and an apparently unstyled `<Heading />` component (see screenshot).

Additional stacktrace information:
```__stack_frame_overlay_proxy_console__
index.js:2178
generateComponentTheme
registry.js:335
generateThemeForContextKey
themeable.js:122
componentWillMount
themeable.js:143
...errors in react-dom.development.js
```

![create-react-app-instructure-ui](https://user-images.githubusercontent.com/1794859/42186759-d094a742-7e1b-11e8-8e1e-ee8be66b9404.PNG)

Thanks for any help!

**Comment by claydiffrient** [comment]
Tommy!  I'm 99% positive you need a theme... there's the Canvas theme out of the box you could use to start then perhaps copy it over and modify it as a template for creating a new theme for your application.  

See https://instructure.design/#canvas at the bottom... the tl;dr is you need something like:

```js
import theme from '@instructure/ui-themes/lib/canvas'
theme.use()
```

before you mount your root component.

**Comment by tgroshon** [comment]
@claydiffrient You're right on good buddy!  I actually just found this like 5 mins ago and when I came back here to update the issue you said to do the exact same thing; I guess it must be right lol.

So I learned a few things:

1. Must have themes; hard-requirement
2. Using the canvas theme as stated is how to get the desired "out-of-the-box" styles shown in examples

Does that sound accurate?  If so, some docs would be AWESOME.  Maybe a really short `Quick Start` section in the `README.md` explaining those two things.  The information there seems to be 90% geared towards developing **on** the library rather than **with** the lib (it's like you guys need to do work or something?!).
